platform/platform-impl/src/com/intellij/openapi/wm/impl/customFrameDecorations/header/toolbar/ExpandableMenu.kt

```diff
@@ -70,12 +70,21 @@ internal class ExpandableMenu(
         updateBounds()
       }
     })
+    if (isSticky()) {
+      SwingUtilities.invokeLater {
+        switchState();
+      }
+    }
   }
 
   fun isEnabled(): Boolean {
     return !SystemInfoRt.isMac && Registry.`is`("ide.main.menu.expand.horizontal")
   }
 
+  fun isSticky(): Boolean {
+    return isEnabled() && !SystemInfoRt.isMac && Registry.`is`("ide.main.menu.expand.horizontal.sticky")
```

Comment:
It would be useful for macOS. There is a registry keys to force IntelliJ one menu on this OS (`apple.laf.useScreenMenuBar` or `jbScreenMenuBar.enabled` I can't remember which one). It is useful for automated UI tests